src/checkers/ast.rs

```diff
@@ -1231,6 +1231,28 @@ where
                     }
 
                     if let Some(asname) = &alias.node.asname {
+                        let full_name = helpers::format_import_from_member(
+                            level.as_ref(),
+                            module.as_deref(),
+                            &alias.node.name,
+                        );
```

Comment:
Nit: let's move this into the `if` below, so that we're not creating this string unless it's needed by the `check_conventional_import` call.